.env.example

```diff
@@ -132,3 +132,9 @@ EMAIL_SERVER_PASSWORD='<office365_password>'
 
 # Set the following value to true if you wish to enable Team Impersonation
 NEXT_PUBLIC_TEAM_IMPERSONATION=false
+
+# Close.com internal CRM
+CLOSECOM_API_KEY=
+
+# Sendgrid internal email sender
+SENDGRID_API_KEY=
```

Comment:
These are needed if you want to hook up Sendgrid and Close.com Sync Services. It remains optional for any self-hoster. Documentation will be done to explain Sync Services and its purpose.